src/data/actions/grades.js

```diff
@@ -244,6 +264,24 @@ const fetchBulkUpgradeHistory = courseId => (
     }).catch(() => dispatch(bulkHistoryError()))
 );
 
+const updateGradesIfAssigGradeFiltersSet = (
```

Comment:
typo: "Assign"

Reply:
Assign is a word in it's own right; I actually find that abbreviation less readable b/c I often misread it as a verb. If this is important to you for consistency's sake then I can do it, but could you confirm

Reply:
Hmm, fair point.  I don't want you to change it to something that you have pointed out a valid shortcoming of.  Is it worth using the whole word instead?  We're only saving 5 characters.

Reply:
Also, this is probably less of a worry in local variable names than it is for function names, so don't feel like you have to track down every instance of "assig" :)

Reply:
Sounds great; I'll change all the abbev.s floatin' arou